### ISSUE TITLE
FileWatcher - Bug Report Consolidation

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#313]
**Describe the bug in current situation**
This ticket will be used to record and track instances where filewatcher conversions are not occuring as expected and/or issues with the log file. 

**Link bug to the User Story**

**Impact of this bug**
Anticipate this will be a collection of edge cases for monitoring and determining future direction based on value. 

**Chance of Occurring (high/medium/low/very low)**
Due to nature of excel files, difficult to assess frequency at this time.

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**
File folder names will be listed where problem conversions have been noted.

**Steps to Reproduce**
If possible, include reproduction steps in comment replies 

**Actual/ observed behaviour/ results**

**Expected behaviour**
A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by m-prodan** [comment]
\ARR\In_Coming\IRR-2021-12470 - Judy Law\(Filewatcher - Do Not Use) 

Good morning,
Many of these excel files in this records response failed according to the FW log.
Just passing along for your FYI.

Note from Matt: I have access to this folder and attempted to replicate. Creating a new folder and moved the failed files into it - conversion did occur when I did this, however, not all tabs printed, and log deemed a fail. I suspect there is an issue with the functions on those specific tabs. There was no error message when excel file was opened, but a general warning about 'external connections' that I wonder may be having an impact on converting.

**Comment by abin-aot** [comment]
Logged an issue with Syncfusion , https://www.syncfusion.com/support/directtrac/incidents/339258

**Comment by abin-aot** [comment]
@m-prodan The 3 problematic excel files stated above is getting converted now. We got a hot fix from syncfusion to handle the difficult scenarios within the  various excel sheets. Please try out

**Comment by m-prodan** [comment]
@abin-aot - it looks like it worked for the most part; however, I looked at each output sheet and found one issue:

Regarding file 00000024_FY17 - 22 Historical IFE Spending.xlsx - the worksheet tab FY17 Summary printed a blank page. All the other worksheets worked though. Might be an edge case though, as everything else appeared to convert correctly.

I am going to close this ticket, I did get another report from the business just now but I will investigate first before I write up another issue.
